CoreUtilities/src/au/gov/asd/tac/constellation/utilities/text/LanguagetoolClassLoader.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2010-2024 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.utilities.text;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ *
+ * @author Auriga2
+ */
+
+public class LanguagetoolClassLoader {
+    private static URLClassLoader classLoader;
+    public static LanguagetoolClassLoader LanguagetoolClassLoader;
+    public static Class<?> JLanguagetool;
+    public static Class<?> Language;
+    public static Class<?> Languages;
+    public static Class<?> MultiThreadedJLanguageTool;
+    public static Class<?> RuleMatch;
+    public static Class<?> Rule;
+    public static Class<?> SpellingCheckRule;
```

Comment:
these variables should all start lowercase. Also can they be made private with getter functions